SocialCareCaseViewerApi/V1/Boundary/Requests/CreateCaseSubmissionRequest.cs

```diff
@@ -0,0 +1,33 @@
+using System.Text.Json.Serialization;
+using FluentValidation;
+
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+
+#nullable enable
+{
+    public class CreateCaseSubmissionRequest
+    {
+        [JsonPropertyName("formId")]
+        public int FormId { get; set; }
+
+        [JsonPropertyName("socialCareId")]
+        public int ResidentId { get; set; }
```

Comment:
Are we using `ResidentId` because that's what we're currently using in MongoDB? Just wondering if we should name this `PersonId`.

Reply:
We can use any name we want here, I like ResidentId as it distinguishes that the Id is about an individual resident.

A PersonId to me is a little vague about what the Person represents, could easily be an Id to lookup a Worker.